src/main/i18nManager.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {ipcMain} from 'electron';
+import log from 'electron-log';
+
+import {GET_AVAILABLE_LANGUAGES, GET_LANGUAGE_INFORMATION} from 'common/communication';
+
+import {Language, languages} from '../../i18n/i18n';
+
+export function t(s: string, defaultString = '', values: any = {}) {
```

Comment:
I didn't realize when we were discussing this before that you meant `t` was the function that was actually doing the translation. While I don't like `localizeMessage` since we use it all over the place in the web app where `react-intl` is actually available, I guess that'd be a good name for it, and it should be picked up by mmjstool properly. In mobile and web, `t` is just a marker function that doesn't actually do anything, so I don't think we should have it work differently here.

Also, I'm now realizing that more of this translation stuff would probably make a good shared library between the web app and other repos...